code/game/antagonist/station/renegade.dm

```diff
@@ -49,9 +62,22 @@ var/datum/antagonist/renegade/renegades
 		return
 
 	var/gun_type = pick(spawn_guns)
-	var/obj/item/gun = new gun_type(get_turf(player))
-	player.put_in_hands(gun)
+	if(islist(gun_type))
+		gun_type = pick(gun_type)
 
+	var/obj/item/gun = new gun_type(get_turf(player))
+	var/list/slots = list (
+		"backpack" = slot_in_backpack,
+		"left hand" = slot_l_hand,
+		"right hand" = slot_r_hand,
+		)
+	for(var/slot in slots)
+		player.equip_to_slot(gun, slot)
+		if(gun.loc == player)
+			break
+	var/obj/item/weapon/storage/S = locate() in player.contents
```

Comment:
Not really a problem though, is it? Loadout already shoves stuff into medkits and suchlike, and it's better it goes into a medkit than into their hand on spawn.